GoogleSignIn/Sources/GIDEMMErrorHandler.h

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <TargetConditionals.h>
```

Comment:
Is there a reason to use `#include` over `#import` here and throughout?

Reply:
I am following conventions in Google 3 and other libraries. Do you think #import is better here?

Reply:
This specific header is a C header so `#include` is appropriate.

Reply:
In reading up on this, `#import` should be fine and perhaps even helpful as it obviates the need for include guards. In general, using `#include` may entail a small performance hit in compilation as it requires that the header is opened and the include guards (if present) are read and respected. Thus, using `#include` ensures that the importer respects the author's intention at the potential cost of performance. 

Nonetheless, it doesn't seem like an issue in this case as the header does have an include guard. :)